articles/libraries/lock-android/v1/index.md

```diff
@@ -9,6 +9,10 @@ mobileimg: media/articles/libraries/lock-android.png
 
 <%= include('../_includes/_lock-version') %>
 
+::: note
+Check out the [Lock.Android repository](https://github.com/auth0/Lock.Android) on GitHub.
```

Comment:
I think that this should point here https://github.com/auth0/Lock.Android/tree/v1

cc @jeffreylees